semcore/select/src/InputSearch.jsx

```diff
@@ -102,12 +101,8 @@ const SearchValue = (props) => {
 };
 const SearchClear = (props) => {
```

Comment:
The Hint is squished horizontally
<img width="409" alt="image" src="https://github.com/user-attachments/assets/784f75d6-03fb-4691-babf-59d9d01bf045">

Reply:
Let's come back to this later (it's not part of Select, it's something like popper/tooltips). And it's not all trivial.